### DISCUSSION TITLE
Yamcs-Studio displays support in web interface

### DISCUSSION THREAD

**Discussion by guidoSNT** [yamcs/yamcs#850] [General]
Hi, I've been using Yamcs-Studio displays to visualize telemetry and even thought it works great, the web doesn't have all the features. Are there any plans to get full support of OPI displays, or should I keep using the Studio's Runner?

Thanks!

**Comment by fqqb** [comment]
If you can isolate to a specific funtionality that you are missing, please open an issue here: https://github.com/yamcs/opi.js
We add/fix stuff as-needed.
